tests/test_profiling_utils.py

```diff
@@ -10,14 +10,20 @@
 from io import StringIO
 from unittest.mock import patch
 
+import pytest
+from torch import cuda
+
 from habitat_sim.utils import profiling_utils
 
 _ENV_VAR_NAME = "HABITAT_PROFILING"
 
 # Based on the env var, reloading the profiling_utils module should set
 # profiling_utils._enable_profiling to True or False.
+@pytest.mark.skipif(
```

Comment:
There all identical, you can define the skip if once and then use it everywhere
```
requires_cuda =  pytest.mark.skipif( ...
```
```
@requires_cuda
```

Reply:
Message is different, but otherwise yeah, these are the same check.